src/inference/src/os/lin/lin_system_conf.cpp

```diff
@@ -80,7 +80,7 @@ CPU::CPU() {
                 }
                 std::string cache_info;
                 std::getline(cache_file, cache_info);
-                one_info[n] = cache_info;
```

Comment:
There is more like this pattern in this file. Check other if should be corrected also.

Reply:
done